src/cocotb/_ANSI.py

```diff
@@ -0,0 +1,63 @@
+# Copyright cocotb contributors
+# Copyright (c) 2013, 2018 Potential Ventures Ltd
+# Copyright (c) 2013 SolarFlare Communications Inc
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from cocotb._py_compat import StrEnum
+
+_ESCAPE = "\033["
+
+
+class ANSI(StrEnum):
+    """ANSI escape codes for coloring output.
+
```

Comment:
```suggestion
    The color names supported are ``[BRIGHT_]{BLACK|RED|GREEN|YELLOW|BLUE|MAGENTA|CYAN|WHITE}{_FG|_BG}``.

```
Listing them all separately would probably be too much.